brave-spancollector-kafka/src/test/java/com/github/kristofa/brave/kafka/KafkaTestServer.java

```diff
@@ -0,0 +1,65 @@
+package com.github.kristofa.brave.kafka;
+
+import kafka.server.KafkaConfig;
+import kafka.server.KafkaServer;
+import kafka.utils.MockTime;
+import kafka.utils.TestUtils;
+import kafka.utils.TestZKUtils;
+import kafka.utils.ZKStringSerializer$;
+import kafka.zk.EmbeddedZookeeper;
+import org.I0Itec.zkclient.ZkClient;
+import scala.collection.JavaConversions;
+import scala.collection.mutable.Buffer;
+
+import java.util.Collections;
+import java.util.Properties;
+
+public class KafkaTestServer {
```

Comment:
non-blocking note:
maybe just me, but I think `com.github.charithe.kafka.KafkaJunitRule` will lead to less clutter. It is hard to see what these tests are doing. If it is too much work, I could help later, but unfortunately not likely in the next few days.

Ex. It should be much cleaner than this, to use the rule in java: https://github.com/openzipkin/zipkin/blob/master/zipkin-receiver-kafka/src/test/scala/com/twitter/zipkin/receiver/kafka/KafkaProcessorSpec.scala